server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -1115,14 +1115,12 @@ export class AgenticChatController implements ChatHandlers {
             return createAuthFollowUpResult(authFollowType)
         }
 
-        const backendError = err.cause
-        // Send the backend error message directly to the client to be displayed in chat.
-        return {
+        return new ResponseError<ChatResult>(LSPErrorCodes.RequestFailed, err.cause.message, {
```

Comment:
will the error cause message be logged and then the other body be shown to the user?

Reply:
Yeah, see the screenshot I added above. I was debating try to parse the requestId and include it in the chat message (done on the agentic-chat vsc branch), but I think it makes more sense to be the logs since its a technical detail and only useful for internal users. 

I think ideally we add a button to this chat message that opens the logs (like we do in most VSC error messages). However, this is not a p0 to me since it requires a decent amount of work to allow Flare to open IDE logs.